src/data/concerts.yaml

```diff
@@ -255,3 +255,10 @@
   locationLink: "https://youtu.be/czwuJ2rG0gc"
   time: "14:30"
   facebookEvent: "https://www.facebook.com/events/591078945163025"
+
+- id: krashko-13-feb-2021
```

Comment:
Was er geen concertdata voor kako in januari? Niet per se iets wat hier gefixt moet worden, was gewoon benieuwd

Reply:
Nee, die is nooit gemaakt